packages/perseus-editor/src/widgets/plotter-editor.tsx

```diff
@@ -332,7 +495,7 @@ class PlotterEditor extends React.Component<Props, State> {
                 )}
                 <div>
                     Editing values:{" "}
-                    {["correct", "starting"].map((editing) => (
+                    {["correct", "starting"].map((editing: EditingStates) => (
```

Comment:
I think our convention is to name enums as singular: `EditingState`.  This also might be a good candidate for a [TS-array enum](https://khanacademy.slack.com/archives/C028T2HDX3P/p1684430521976579).